pkgs/tools/misc/shim/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchFromGitHub, lib, elfutils, vendorCertFile ? null
+, defaultLoader ? null, ... }:
+
+let
+  gnu-efi-src = fetchFromGitHub {
```

Comment:
Do you think we could have this as a passthru derivation or as a dependency so it can be overriden if necessary (or updated automatically)?

Reply:
not entirely sure how you could use it an a passthru, but otherwise I just rewrote it so you could override it like:
```
shim-unsigned.overrideAttrs(old: {
  gnu_efi_src = fetchFromGitHub {
    owner = "rhboot";
    repo = "gnu-efi";
    rev = "c2102ae05917d81ffc7b854003415e575fc90764";
    hash = "sha256-+LTTfpnQrLBt1cqsipTDEc/wZU8SRwHQz8pfovxZU/0=";
  };
})
```

Reply:
I can't just use the gnu-efi as dependency as we need access to the source and a very particular version of it as the API changes.